config_parser_test.cc

```diff
@@ -1,6 +1,17 @@
 #include "gtest/gtest.h"
 #include "config_parser.h"
 
+class NginxConfigParserStringTest : public ::testing::Test {
```

Comment:
Try to set up your sublime text to use spaces instead of hard tabs. This apparently is a convention among many developer communities.